### ISSUE TITLE
.slick-arrows should have a focus outline for accessibility

### ISSUE THREAD

**Issue by Jakobud** [kenwheeler/slick#3327]
.slick-arrows should have an outline when they are focused. Currently in the slick-theme.scss they have outline: none; This is bad for accessibility.

====================================================================

#### Steps to reproduce the problem

1. Focus either the prev or next slick-arrow using the keyboard/tabbing. You will see they have no focus outline so a keyboard-only user would never be able to see that they have them focused. This is bad for accessibility.

Here is the CSS that needs to change:

https://github.com/kenwheeler/slick/blob/master/slick/slick-theme.scss#L82

====================================================================

#### What is the expected behaviour?

You should be able to see the default browser outline when it's focused

====================================================================

#### What is observed behaviour?

There is no outline so it's impossible to see when it is focused

====================================================================

Simply remove this line:

https://github.com/kenwheeler/slick/blob/master/slick/slick-theme.scss#L82

And that would fix the problem.
